app/models/hourglass/settings_validation.rb

```diff
@@ -0,0 +1,84 @@
+module Hourglass
+  class SettingsValidation
+    include ActiveModel::Model
+
+    attr_accessor :global_settings,
+                  :round_sums_only,
+                  :round_minimum,
+                  :round_limit,
+                  :round_default,
+                  :round_carry_over_due,
+                  :report_title,
+                  :report_logo_url,
+                  :report_logo_width,
+                  :global_tracker
+
+    validates :round_minimum, numericality: true
+    validates :round_limit, numericality: { only_integer: true }
+    validates :round_carry_over_due, numericality: true
+    validates :report_title, length: { maximum: 23 }, if: :global_settings
+    validates :report_logo_url, length: { maximum: 23 }, if: :global_settings
+    validates :report_logo_width, numericality: { only_integer: true }, if: :global_settings
+
+    def initialize(settings)
+      settings = parse_boolean [:round_default, :round_sums_only, :global_tracker], settings
+      settings = parse_float [:round_minimum, :round_carry_over_due], settings
+      settings = parse_int [:round_limit, :report_logo_width], settings
+      parse_settings settings
```

Comment:
I think its better to move the parsing all to parse settings. maybe name it from_hash